guide_questions/readme.md

```diff
@@ -1,19 +1,19 @@
 # Guide-questions
 
-It's is important to answer some guide-questions before to start any Data Science project.
+It's important to answer some guide-questions before starting any Data Science project.
 
 1. **Which problem do we wanna solve?** Predict house prices based in more than a hundred of aspects (variables).
```

Comment:
Suggestion: I think it is better to use _Which problem do we **WANT TO** solve?_.

Reply:
done